deps/3rdparty/libdispatch/build/libdispatch.vcxproj

```diff
@@ -249,19 +246,13 @@
     <ClangCompile Include="..\src\shims\mach.c" />
   </ItemGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
-
   <PropertyGroup>
-    <PrebuiltPath Condition="'$(PrebuiltPath)' == ''">$(WINOBJC_SDK_ROOT)\deps\prebuilt\Universal Windows\$(PlatformTarget)\$(Configuration)\\</PrebuiltPath>
+    <PrebuiltRoot Condition="'$(PrebuiltRoot)' == ''">$(WINOBJC_SDK_ROOT)\tools\deps\prebuilt\\</PrebuiltRoot>
```

Comment:
>\\ [](start = 93, length = 2)

double backslash?

Reply:
Msbuild requirement. A dangling \ will eat an adjacent “ in the compiler command line or the command line for any other tool.

Reply:
I was merely copying the double backslash from the changed line. In retrospect, we don't need this anyway. The target that used it was removed.